minecraft.js

```diff
@@ -66,6 +66,25 @@ app.post('/command', (req, res) => {
   }, 250);
 });
 
+app.get('/properties', (req, res) => {
+  const text = fs.readFileSync('./server.properties').toString('utf-8');
+  const result = fileToJSON(text);
+  res.send(result);
+});
+
+app.put('/properties', (req, res) => {
+  const properties = req.body;
+  let string = '#Minecraft server properties\n#(last boot timestamp)\n';
+  for (const property in properties) {
+    const sub_string = `${property}=${properties[property]}\n`;
+    string = string.concat(sub_string);
+  }
+  fs.writeFile('./server.properties', string, function (err) {
+    if (err) return console.log(err);
+  });
+  res.send({ status: 'server.properties successfully modified!' });
```

Comment:
I'd return a status 204 no content and not return any body.

Reply:
Great idea! I've made that change.